pre_commit/main.py

```diff
@@ -34,11 +34,19 @@ def _add_color_option(parser):
     )
 
 
+def _add_config_option(parser):
+    parser.add_argument(
+        '-c', '--config', default='.pre-commit-config.yaml',
+        help='Path to alternate config file'
+    )
+
+
 def main(argv=None):
     argv = argv if argv is not None else sys.argv[1:]
     argv = [five.to_text(arg) for arg in argv]
     parser = argparse.ArgumentParser()
 
+    parser.set_defaults(config='.pre-commit-config.yaml')
```

Comment:
I'm pretty sure Runner doesn't get constructed in the help case (otherwise my color stuff would be broken)